opt/heroku-metrics-daemon.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# TODO set -eu
```

Comment:
This looks different from https://github.com/heroku/heroku-buildpack-metrics/blob/master/.profile.d/heroku-metrics-daemon.sh

Do we no longer need to install agentmon in the buildpack?

Reply:
Correct. not for java apps